packages/manager/modules/dedicated-universe-components/src/config/index.js

```diff
@@ -0,0 +1,16 @@
+import { Environment } from '@ovh-ux/manager-config';
+import constantsConfig from './constants.config';
+
+const aapiRootPath = '/engine/2api/';
+const swsProxyRootPath = 'apiv6/';
+
+const target = Environment.getRegion();
+// const prodMode = !!WEBPACK_ENV.production;
```

Comment:
If webpack variable are used maybe we should keep those declaration in the app

Reply:
yes, I will add this declaration into the app.